src/senaite/core/browser/stickers/templates/stickers/QR_1x14mmx39mm.pt

```diff
@@ -5,7 +5,7 @@
   <!-- QR Code -->
   <div class="qrcode"
        data-size="36"
-       tal:attributes="data-code sample_id">
+       tal:attributes="data-text python:view.get_id(item);">
```

Comment:
Got this:

```
AttributeError: 'StickersView' object has no attribute 'get_id'
```

You might consider to use the `sample_id` variable, initialized at line 2

Reply:
Apologies, I was doing this in a custom codebase and did not see that I was on the extended stickers class.
If fixed it in https://github.com/senaite/senaite.core/pull/2708/commits/f4f29653b8406ccac1f808036cb7350dfd92ca60